eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -31,16 +31,16 @@ function ToSemVer($version){
 
             # some older packages don't have a prenumber, should handle this
             if($matches["prenumber"]){
-                $prenumber = [int]$matches["prenumber"]
+                $prenumber = [long]$matches["prenumber"]
             }
 
             $isPre = $true;
         }
 
         New-Object PSObject -Property @{
```

Comment:
We should remove this and use our SemVer type instead.

Reply:
Looking a little closer at the failure I suspect if you switch this to our SemVer type it will fix the error.